frontend/packages/process-editor/src/components/ConfigPanel/ConfigPanel.test.tsx

```diff
@@ -37,124 +36,89 @@ const mockBpmnContextValue: BpmnContextProps = {
+      name: textMock('process_editor.configuration_panel_change_task_id'),
+    });
+    expect(editTaskIdButton).toBeInTheDocument();
   });
 
   it('should display the details about the end event when bpmnDetails.type is "EndEvent" and customizeEndEvent feature flag is enabled', () => {
-    jest.spyOn(FeautreToggleUtils, 'shouldDisplayFeature').mockReturnValue(true);
-    render({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.EndEvent } });
-    expect(
-      screen.getByText(textMock('process_editor.configuration_panel_end_event')),
-    ).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "data" task is selected', () => {
-    render();
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_data_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "confirmation" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'confirmation' } });
+    (shouldDisplayFeature as jest.Mock).mockReturnValue(true);
+    renderConfigPanel({ bpmnDetails: { ...mockBpmnDetails, type: BpmnTypeEnum.EndEvent } });
 
     expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_confirmation_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
-  });
-
-  it('should display the details about the selected task when a "feedback" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'feedback' } });
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_feedback_task'),
-        level: 2,
-      }),
+      screen.getByText(textMock('process_editor.configuration_panel_end_event')),
     ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
   });
 
-  it('should display the details about the selected task when a "signing" task is selected', () => {
-    render({ bpmnDetails: { ...mockBpmnDetails, taskType: 'signing' } });
-
-    expect(
-      screen.getByRole('heading', {
-        name: textMock('process_editor.configuration_panel_signing_task'),
-        level: 2,
-      }),
-    ).toBeInTheDocument();
-
-    expect(screen.getByText(mockBpmnDetails.id)).toBeInTheDocument();
-    expect(screen.getByText(mockBpmnDetails.name)).toBeInTheDocument();
+  it.each([
+    {
+      task: BpmnTypeEnum.Process,
+      expectedText: 'process_editor.configuration_panel_no_task_title',
+    },
```

Comment:
Would it be an option having a separate test for this case so the text-key: `process_editor.configuration_panel_element_not_supported_message` can be hardcoded for this each-test to make it a bit shorter and easier to read? 😊

Reply:
I did that at first but then I changed my mind to do it this way instead. The reason was that I found it quite repetitive to implement the same code over and over again. So instead of re-writing the same test repeatedly, I structure a test data instead.